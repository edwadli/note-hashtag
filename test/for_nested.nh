for n in { "a: " "b: " "c: " } do (
  Print n
  for n in { 1 2 3 } do (
    Print (StringOfInt n)
  )
  Print "\n"
)

for n in { 10 100 } do for m in { 1 2 3 } do PrintEndline (StringOfInt (n * m))
