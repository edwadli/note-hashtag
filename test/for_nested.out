a: 123
b: 123
c: 123
10
20
30
100
200
300
